powerVCF.psm1

```diff
@@ -3149,6 +3149,98 @@ Function Get-VCFUser
   }
 }
 Export-ModuleMember -Function Get-VCFUser
+Function New-VCFUser
+  {
+    <#
+      .SYNOPSIS
+      Connects to the specified SDDC Manager and adds a new user.
+
+      .DESCRIPTION
+      The New-VCFUser cmdlet connects to the specified SDDC Manager and adds a new user with a specified role.
+
+      .EXAMPLE
+      PS C:\> New-VCFUser -user sec-admin@rainpole.io -role ADMIN
```

Comment:
Consider a different user example for this function maybe vcf-admin@rainpole.io ?

Reply:
Updated in doc examples also